scripts/setup.sh

```diff
@@ -2,10 +2,22 @@
 
 mongo mongodb://localhost:27017/codelists <<EOF
 db.dropDatabase();
+db.codelists.ensureIndex({"id":1},{"background":true});
 db.codes.ensureIndex({"links.code_list.id":1},{"background":true});
 db.codes.ensureIndex({"code":1,"links.code_list.id":1},{"background":true});
 EOF
 
 scriptDir=$( dirname "${BASH_SOURCE[0]}" )
-mongoimport --db codelists --collection codelists --file $scriptDir/codelists.json
-mongoimport --db codelists --collection codes --file $scriptDir/codes.json
+cd $scriptDir || exit 2
+
+import_to() {
+        local collection=$1 file=$2; shift 2
+        echo Importing $file ...
+        mongoimport --db codelists --collection $collection --file $file || exit 2
```

Comment:
It maybe worth allowing the mongodb URI to be configured here

Reply:
maybe a feature worth adding when it's needed? dont know that it's needed right now